hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/SerializationUtilTest.java

```diff
@@ -102,6 +109,17 @@ public void testReadWriteNullableBoolean_whenInvalid()
         deserialize(bytes);
     }
 
+    @Test
+    public void testCreateSerializerAdapter() {
+        ClassNameFilter classNameFilter = new SerializationClassNameFilter(new JavaSerializationFilterConfig());
+        // JavaSerializer is instance of StreamSerializer, hence using it as parameter
+        SerializerAdapter streamSerializerAdapter = SerializationUtil.createSerializerAdapter(new JavaSerializer(true, true, classNameFilter));
```

Comment:
```suggestion
        // ArrayStreamSerializer is instance of StreamSerializer, hence using it as parameter
        SerializerAdapter streamSerializerAdapter = SerializationUtil.createSerializerAdapter(new ArrayStreamSerializer());
```
use this instead to simplify the test

Reply:
@mdumandag  Hi, I have made the changes for this and updated this pull request.